### PR TITLE
wip: apply new jsdoc stylilng

### PR DESCRIPTION
**Pull request by RobinBol**
Screenshot 2021-05-10 at 11 04 12

### CODE REVIEW

**Comment by RobinBol** [comment]
@tjallingt do you know what these "Externals" are (in the side bar on the left)?

File: docs/jsdoc.json [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-    "referenceTitle": "documentation homey-zigbeedriver"
+    "default": {
+      "outputSourceFiles": false
+    }
   },
-  "opts": {
-    "destination": "../build/docs",
-    "template": "../node_modules/braintree-jsdoc-template"
+  "tsUtils": {
+    "typeScriptUtilityTypes": false
```
Comment (tjallingt) [inline docs/jsdoc.json:21]: This should prevent the externals from showing up I believe... not sure why it isn't working

File: docs/jsdoc.json [hunk 2]
```diff
@@ -1,20 +1,23 @@
 {
+  "source": {
+    "include": ["./lib", "./index.js", "./lib/util"]
+  },
+  "opts": {
+    "mainpagetitle": "Homey ZigbeeDriver",
+    "template": "./node_modules/homey-jsdoc-template",
+    "readme": "./README.md",
+    "destination": "./build/docs"
```
Comment (tjallingt) [inline docs/jsdoc.json:9]:
```suggestion
    "destination": "./build"
```

File: package.json [hunk 3]
```diff
@@ -6,10 +6,10 @@
   "scripts": {
     "test": "mocha --recursive",
     "lint": "eslint .",
-    "serve": "concurrently \"serve build/docs/\" \"npm run jsdoc:watch\"",
-    "build": "npm run jsdoc",
-    "jsdoc": "cd docs; jsdoc -c jsdoc.json",
-    "jsdoc:watch": "npm-watch jsdoc"
+    "serve": "concurrently \"serve build/docs/\" \"npm run build:watch\"",
```
Comment (tjallingt) [inline package.json:9]:
```suggestion
    "serve": "concurrently \"serve build/\" \"npm run build:watch\"",
```

**Pull request merged**